src/current/cockroachcloud/serverless-faqs.md

```diff
@@ -27,25 +27,25 @@ To get started with CockroachDB {{ site.data.products.serverless }}, <a href="ht
 
 All non-contract CockroachDB {{ site.data.products.cloud }} organizations are given 50M RUs and 10 GiB of storage for free each month. Free resources do not apply to contract customers. Free resources can be spent across all CockroachDB {{ site.data.products.serverless }} clusters in an organization and will appear as a deduction on your monthly invoice.
 
-[Setting higher resource limits]({% link cockroachcloud/serverless-cluster-management.md %}#edit-cluster-capacity) will allow your cluster to scale to meet your application's needs and maintain a high level of performance. If you reach your storage limit, your cluster will be throttled and you will only be able to delete data or increase your storage limit. If you reach your RU limit, your cluster will be disabled until the end of the billing cycle unless you increase your RU limit.
+[Setting higher resource limits]({% link cockroachcloud/basic-cluster-management.md %}#edit-cluster-capacity) will allow your cluster to scale to meet your application's needs and maintain a high level of performance. If you reach your storage limit, your cluster will be throttled and you will only be able to delete data or increase your storage limit. If you reach your RU limit, your cluster will be disabled until the end of the billing cycle unless you increase your RU limit.
 
 Organizations without billing information on file can only create one cluster. Once you [set up billing information]({% link cockroachcloud/billing-management.md %}), your organization can have up to 200 clusters.
 
 ### What is a Request Unit?
 
-With CockroachDB {{ site.data.products.serverless }}, you are charged for the storage and activity of your cluster. All cluster activity, including SQL queries, bulk operations, and background jobs, is measured in [Request Units]({% link cockroachcloud/plan-your-cluster-serverless.md %}#request-units), or RUs. An RU is an abstracted metric that represent the size and complexity of requests made to your cluster. See [Plan your CockroachDB {{ site.data.products.serverless }} Cluster]({% link cockroachcloud/plan-your-cluster-serverless.md %}#request-units) for more information.
+With CockroachDB {{ site.data.products.serverless }}, you are charged for the storage and activity of your cluster. All cluster activity, including SQL queries, bulk operations, and background jobs, is measured in [Request Units]({% link cockroachcloud/plan-your-cluster-basic.md %}#request-units), or RUs. An RU is an abstracted metric that represent the size and complexity of requests made to your cluster. See [Plan your CockroachDB {{ site.data.products.serverless }} Cluster]({% link cockroachcloud/plan-your-cluster-basic.md %}#request-units) for more information.
```

Comment:
Nit: Update the variable?

Reply:
No, I had to leave the variables as is since I'm not updating this file in this PR. Only changing the links for now. I know it's messy, but I'd rather look at each file holistically than change them piecemeal in multiple PRs